style.css

```diff
@@ -161,7 +162,256 @@
     display: flex;
   }
 
+  h2 {
+    text-align: center;
+    color: black;
+    margin-top: 120px;
+    width: 100%;
+    margin-left: 24px;
+    font-family: "Crete Round", serif;
+    font-size: 40px;
+    line-height: 24px;
+    font-weight: 400;
+    height: 123px;
+    max-width: fit-content;
+  }
+
+  .image1 {
+    height: 237px;
+    width: 100%;
+    margin-left: 24px;
+    margin-top: 80px;
+  }
+
+  .vector {
+    margin-left: 24px;
+    width: 100%;
+    max-width: fit-content;
+    margin-top: -50px;
+  }
+
+  .texts1 {
+    height: 36px;
+    width: 100%;
+    margin-top: 24px;
+    border-radius: nullpx;
+    font-family: "Crete Round", serif;
+    font-size: 32px;
+    font-weight: 400;
+    line-height: 44px;
+    text-align: left;
+    margin-left: 24px;
+    color: #091e42;
+  }
+
+  .texts2 {
+    height: 147px;
+    width: 100%;
+    margin-left: 24px;
+    margin-top: 12px;
+    font-family: "Inter", serif;
+    font-size: 17px;
+    font-weight: 400;
+    line-height: 24px;
+    text-align: left;
+    max-width: fit-content;
+    color: #091e42;
+  }
+
+  .tags {
+    height: 40px;
+    width: 314px;
+    margin-left: 24px;
+    margin-top: 12px;
+  }
+
+  .see {
+    height: 48px;
+    width: 124px;
+    margin-left: 10px;
+    margin-top: 12px;
+    padding: 12px;
+  }
+
+  .grid-container {
+    display: grid;
+    grid-template: repeat(6, 386px) / repeat(1, 327px);
+    row-gap: 24px;
+  }
+
+  .first,
+  .second,
+  .third,
+  .fourth,
+  .fifth,
+  .sixth {
+    background-image: url("icons/background.png");
+    max-width: fit-content;
+    width: 327px;
+    height: 386px;
+    margin-left: 24px;
+    margin-top: 30px;
+  }
```

Comment:
You have applied a fixed width of 327px on lines 238 and 250, (and many other parts of the page). This prevents your website from being responsive and expanding when the width is increased thereby creating a big blank space to the right as shown below. Kindly remove all the fixed widths and ensure that your design is responsive and the width increases when the screen size increase. 


![Screenshot from 2023-04-29 07-54-18](https://user-images.githubusercontent.com/65233860/235291613-71736a14-a308-4ae3-a0a1-c5c35b6577ae.png)